### PR TITLE
feat(mobile): add google auth component

### PR DESCRIPTION
**Pull request by Kiya971**
add google auth signin button component

### CODE REVIEW

File: mobile/android/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1,3 +1,4 @@
 <resources>
     <string name="app_name">mobile</string>
+     <string name="web_client_id">807234752761-92gmhcgop4pcqflhdiftfhtt4bdhr3a4.apps.googleusercontent.com</string>
```
Comment (Pablodeibar) [inline mobile/android/app/src/main/res/values/strings.xml:3]: is that normal that the client id is not an env variable ?

**Review by AlexGuillard — DISMISSED** [review]
ok

**Review by AlexGuillard — APPROVED** [review]
ok

File: mobile/android/app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Miou-zora) [reply mobile/android/app/src/main/res/values/strings.xml]: +1

**Review by AlexGuillard — APPROVED** [review]
ok

**Review by Benjicatch — APPROVED** [review]

**Pull request merged**